mta/entities/analysis_results.py

```diff
@@ -30,7 +36,18 @@ def is_displayed(self):
     def clear_search(self):
         """Clear search"""
         if self.search.value:
-            self.search.fill("")
+            self.close_search.click()
+
+    @ParametrizedView.nested
+    class AnalysisRowView(ParametrizedView):
```

Comment:
Optional:

If you use normal name then it will be great as its parameterized view for consistency

```
@ParametrizedView.nested
class analysis_row(ParametrizedView): #NOQA
    pass
```